File: src/main/java/com/sfh/pokeRogueBot/model/modifier/impl/MoneyRewardModifierItem.java
package com.sfh.pokeRogueBot.model.modifier.impl;

import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class MoneyRewardModifierItem extends ModifierItem {

    public static final String TARGET = "MoneyRewardModifierType";

    private float moneyMultiplier;
    private String moneyMultiplierDescriptorKey;
}

File: src/main/kotlin/com/sfh/pokeRogueBot/model/modifier/impl/MoneyRewardModifierItem.kt
package com.sfh.pokeRogueBot.model.modifier.impl

class MoneyRewardModifierItem : ModifierItem() {

    companion object {
        const val TARGET = "MoneyRewardModifierType"
    }

    var moneyMultiplier: Float = 0f
    var moneyMultiplierDescriptorKey: String? = null
}